src/sparse_int_set.jl

```diff
@@ -210,7 +210,8 @@ struct ZippedSparseIntSetIterator{VT,IT}
     end
 end
 
-Base.zip(s::SparseIntSet...;kwargs...) = ZippedSparseIntSetIterator(s...;kwargs...)
+Base.zip(s0::SparseIntSet, s::SparseIntSet...; kwargs...) =
+    ZippedSparseIntSetIterator(s0, s...; kwargs...)
```

Comment:
If we are doing multiline we should do multipline

```suggestion
function Base.zip(s0::SparseIntSet, s::SparseIntSet...; kwargs...)
    return ZippedSparseIntSetIterator(s0, s...; kwargs...)
end
```

we do need to officially adopt a style guide #539